include/meltpooldg/flow/surface_tension_operation.hpp

```diff
@@ -14,6 +14,26 @@ namespace MeltPoolDG::Flow
 {
   using namespace dealii;
 
+  /**
+   *  This class enables to compute the contribution to interfacial forces due
+   *  to temperature-(in)dependent surface tension forces
+   *
+   *          /                \        /                         \
+   *  f_st =  | N_a,  α κ n  δ |   +    | N_a,  (I - n ⊗ n ) ∇α δ |
+   *          \            Γ   /        \             Γ   Γ       /
+   *                            Ω                                  Ω
+   *
+   *  with the temperature-dependent surface tension
+   *
+   *  α = α_0 - α'_0 ( T - T    )
```

Comment:
```suggestion
   *  α = min(α_0, α_0 - α'_0 ( T - T    ))
```
as far as I understand the code.

Reply:
You're right. I've added a note on that.

Reply:
I've clarified this issue.